frontend/lib/parsing/ParserContextImpl.h

```diff
@@ -890,6 +890,17 @@ Identifier* ParserContext::buildEmptyIdent(YYLTYPE location) {
 Identifier* ParserContext::buildIdent(YYLTYPE location, PODUniqueString name) {
   return Identifier::build(builder, convertLocation(location), name).release();
 }
+Identifier* ParserContext::buildAttributeIdent(YYLTYPE location,
+                                               PODUniqueString name) {
+  // remove the leading '@' from the name
+  UniqueString n = name;
+  if (n.startsWith("@")) {
+    // remove the @ sign
+    n = UniqueString::get(context(), n.c_str()+1);
+  }
```

Comment:
I think we should have an `else INT_ASSERT` or something here, since right now we're only relying on the function getting called appropriately to prevent it.  I don't think there's a scenario where we'd want something to be accepted that didn't have a `@` at the beginning, right?

Reply:
I'll add the assertion but this doesn't worry me because it wouldn't get through the parser without a leading `@`